src/main/java/com/oltpbenchmark/benchmarks/otmetrics/procedures/GetSessionRange.java

```diff
@@ -70,6 +72,7 @@ public List<Object[]> run(Connection conn, int source_id, int session_low, int s
                 finalResults.add(arr);
             }
         }
+        stmt.close();
```

Comment:
This isn't Oracle support specific, correct?
Maybe file a separate bug/PR for this?

Reply:
Leaving this would break Oracle run for `otmetrics` because too many open connection due to not calling `stmt.close()` . Do I still remove this change and remove `otmetrics` from the GitHub Actions Oracle run?

Reply:
#396